docs/checkers/index.md

```diff
@@ -2557,6 +2557,16 @@ and `pdb.set_trace()`) are found. These breakpoints should be removed in product
 
 ```
 
+## Modified Iteration
+
+### Not a modifiable dictionary
```

Comment:
1. Use the exact name of the error (but using standard English capitalization of the first word, and spaces instead of underscores).
2. Include the error code as well, following the format of the other message titles.
3. Also, include the error code in parentheses and with a `=` above this header, again following the example of the other messages.